api/ephemeral-access/v1alpha1/accessrequest_types.go

```diff
@@ -173,6 +173,21 @@ func (ar *AccessRequest) UpdateStatusHistory(newStatus Status, details string) {
 	ar.Status = *status
 }
 
+// GetLastStatusDetails will return the last recorded details message in the
+// history associated with the given status.
+func (ar *AccessRequest) GetLastStatusDetails(status Status) string {
+	for i := len(ar.Status.History) - 1; i >= 0; i-- {
```

Comment:
Could do `i := range slices.Backward(ar.Status.History)`

Reply:
That is cool. I would need to update to Go 1.23 first.

Reply:
Ah sure, no worries then